src/main/java/ecdar/presentations/SystemPresentation.java

```diff
@@ -79,8 +78,8 @@ private void initializeFrame() {
         // Generate top right corner (to subtract)
         final Polygon topRightCorner = new Polygon(
                 system.getBox().getWidth(), 0,
-                system.getBox().getWidth() - Grid.CORNER_SIZE - 2, 0,
-                system.getBox().getWidth(), Grid.CORNER_SIZE + 2
+                system.getBox().getWidth() - Ecdar.CANVAS_PADDING * 3 + 8, 0,
```

Comment:
I feel like this makes more sense if canvas padding changes
```suggestion
                system.getBox().getWidth() - Ecdar.CANVAS_PADDING * 4 - 2, 0,
```

Reply:
`CORNER_SIZE` constant introduced for models